README.md

```diff
@@ -23,11 +23,11 @@
   - Clone the [The Plutus repository](https://github.com/input-output-hk/plutus), check out the correct commit
     as specified in [cabal.project](code/week01/cabal.project).
   - Set-up IOHK binary caches [How to set up the IOHK binary caches](https://github.com/input-output-hk/plutus#iohk-binary-cache). "If you do not do this, you will end up building GHC, which takes several hours. If you find yourself building GHC, STOP and fix the cache."
-  - Enter a `nix-shell`.
+  - Enter a `nix-shell` in the Plutus repository directory with the correct commit checked out.
   - Build the [English Auction](code/week01) contract with `cabal build` (you may need to run `cabal update` first).
   - Go to the `plutus-playground-client` folder.
-  - Start the Playground server with `plutus-playground-server`.
-  - Start the Playground client (in another `nix-shell`) with `npm run start`.
+  - Start the Playground server with `plutus-playground-server`. When you will run this for the first time, a `nix` will build some things in the background, this will take time, be patient.
```

Comment:
I think we can drop the word `will` in the first sentence. The following example below reads a little better.

"When you run this for the first time..."

Reply:
Makes sense, updated.